security-openid-connect-web-authentication-quickstart/README.md

```diff
@@ -72,6 +72,8 @@ This command will leave Quarkus running in the foreground listening on port 8080
     - You should have access to a HTML page that shows information based on the ID Token, Access Token and Refresh Token issued
     to the application. Where these tokens are available for injection as you can see in the `TokenResource` JAX-RS Resource.
 
+_NOTE:_ Maven "package" phase requires docker Keycloak Server down. Then make sure it is up with the realm when running the app.
```

Comment:
```suggestion
_NOTE:_ Running the tests with, for instance, `mvn package` requires the Keycloak server to be down as it will launch its own one. However, when running the application, make sure it is up with the realm properly configured.
```